app/form_models/school_form.rb

```diff
@@ -1,10 +0,0 @@
-class SchoolForm < VacancyForm
```

Comment:
We still need this form. https://github.com/DFE-Digital/teacher-vacancy-service/blob/master/app/controllers/hiring_staff/vacancies/schools_controller.rb

Reply:
No we don't?

Reply:
The link is to a file which uses it. This PR doesn't delete that file. So we are still using it. What am I missing?

Reply:
it's `SchoolsForm` not `SchoolForm`